File: 2355-maximum-consecutive-floors-without-special-floors/maximum-consecutive-floors-without-special-floors.java
class Solution {
    public int maxConsecutive(int bottom, int top, int[] special) {
        Arrays.sort(special);
        int m=Math.max(special[0]-bottom,top-special[special.length-1]);
        for(int i=0;i<special.length-1;i++){
            m=Math.max(m,special[i+1]-special[i]-1);
        }
        return m;
    }
}

File: 2355-maximum-consecutive-floors-without-special-floors/maximum-consecutive-floors-without-special-floors.py
class Solution(object):
    def maxConsecutive(self, bottom, top, special):
        special= sorted(special)
        m=max(special[0]-bottom,top-special[len(special)-1])
        for i in range(0,len(special)-1):
            m=max(m,special[i+1]-special[i]-1)
        return m if m!=1 else 0
        